workers/unity/Packages/com.improbable.gdk.core/Worker/Worker.cs

```diff
@@ -163,7 +168,9 @@ private Worker(string workerType, Connection connection, ILogDispatcher logDispa
         public static async Task<Worker> CreateWorkerAsync(
             LocatorConfig parameters,
             ConnectionParameters connectionParameters,
-            ILogDispatcher logger, Vector3 origin)
+            ILogDispatcher logger,
+            Vector3 origin,
+            CancellationToken token)
```

Comment:
Technical this is a breaking change as this is part of the public API.. add a small note to the Changelog about this 😄